### PR TITLE
Fixes NIP-105

### PR DESCRIPTION
**Pull request by larubbio**
We now log the response as well as the request on API calls.

### CODE REVIEW

File: api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java [hunk 1]
```diff
@@ -68,6 +68,17 @@ public void setMaxAllowedEndOfUsagePrompt(int maxAllowedEndOfUsagePrompt) {
...
 
     @Override
+    public String toString() {
+        return "FlwUserResponse{" +
+                "currentUsageInPulses=" + currentUsageInPulses +
+                ", endOfUsagePromptCounter=" + endOfUsagePromptCounter +
+                ", welcomePromptFlag=" + welcomePromptFlag +
+                ", maxAllowedUsageInPulses=" + maxAllowedUsageInPulses +
+                ", maxAllowedEndOfUsagePrompt=" + maxAllowedEndOfUsagePrompt +
+                "} " + super.toString();
```
Comment (frankhuster) [inline api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java]: I never saw a super.toString inside a toString before, cool. and a bit weird.

File: api/src/main/java/org/motechproject/nms/api/web/KilkariController.java [hunk 2]
```diff
@@ -88,7 +88,7 @@
     public InboxResponse getInboxDetails(@RequestParam(required = false) Long callingNumber,
                                          @RequestParam(required = false) Long callId) {
 
-        log("/kilkari/inbox", String.format("callingNumber=%s, callId=%s",
+        log("REQUEST: /kilkari/inbox", String.format("callingNumber=%s, callId=%s",
```
Comment (frankhuster) [inline api/src/main/java/org/motechproject/nms/api/web/KilkariController.java]: If we're marking posts with (POST), should we also mark gets with (GET)?

**Comment by frankhuster** [comment]
lgtm, one suggestion you're welcome to follow or not, then merge yourself.

File: api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java (hunk 1 above)
Comment (larubbio) [inline api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java]: intellij has that as one of the options when you tell it to insert
toString()

File: api/src/main/java/org/motechproject/nms/api/web/KilkariController.java (hunk 2 above)
Comment (larubbio) [inline api/src/main/java/org/motechproject/nms/api/web/KilkariController.java]: One method had that in it's description I removed it and decided to leave
GET of as the default for no good reason.

**Pull request merged**